README.rst

```diff
@@ -2,11 +2,8 @@
 .. |PyPI| image:: https://img.shields.io/pypi/v/pypeit?label=PyPI&logo=pypi&logoColor=white
     :target: https://pypi.org/project/pypeit/
 
-.. |Conda| image:: https://img.shields.io/conda/vn/conda-forge/pypeit?label=conda%20version
-    :target: https://anaconda.org/conda-forge/pypeit
-
-.. |CondaDownloads| image:: https://img.shields.io/conda/dn/conda-forge/pypeit?label=conda%20downloads
-    :target: https://anaconda.org/conda-forge/pypeit
+.. |pypi_downloads| image:: https://img.shields.io/pypi/dm/pypeit
```

Comment:
isn't this redundant with the `PyPI` target defined above?

Reply:
Link target, yes.  Content of badge, no.  Do you have a suggestion for a different link target?

Reply:
ah, i see what it's doing now. for consistency with that the badges show, i would target `|PyPI|` to `pypeit/#history` and `|pypi_downloads|` to `pypeit/#files`. otoh, i don't think it matters all that much so fine to leave as-is as well.

Reply:
@tepickering I didn't see this suggestion until after this PR merged... updated links are included in #1712 .